src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationOptions.cs

```diff
@@ -48,6 +48,19 @@ public void AddScheme(string name, Action<AuthenticationSchemeBuilder> configure
         }
 
         /// <summary>
+        /// Adds an <see cref="AuthenticationScheme"/>.
+        /// </summary>
+        /// <typeparam name="THandler">The <see cref="IAuthenticationHandler"/> responsible for the scheme.</typeparam>
+        /// <param name="name">The name of the scheme being added.</param>
+        /// <param name="displayName">The display name for the scheme.</param>
+        public void AddScheme<THandler>(string name, string displayName) where THandler : IAuthenticationHandler
```

Comment:
Its just Name + DisplayName + Type, I guess its slightly more future proof with the action.